nci/workspace_cleaner.rb

```diff
@@ -82,9 +82,22 @@ def cleanup?(workspace)
         puts '  ws-cleanup => delete'
         return true
       end
+      # delete mgmt jobs workspaces when they become old 
+      # and are phased out - NCI.old_series
+      if NCI.old_series
+        if workspace.include?('mgmt_') && workspace.include?("#{NCI.old_series}")
```

Comment:
"#{NCI.old_series}" is redundant. It creates a new string and then interpolates a variable into it. Fairly costly affair that. It does so using a builtin function .to_s. So this is a slow way to express `NCI.old_series.to_s`. Except that is also redundant because old_series (when defined) is already a string. So really this could just be `workspace.include?(NCI.old_series)`.

You could also opt to compress the if conditions. But that is a matter of taste.

`if NCI.old_series && workspace.include?(NCI.old_series)`

If the first condition is already false the second will not execute so this is safe.